packages/esm-outpatient-app/src/patient-search/patient-scheduled-visits.component.tsx

```diff
@@ -70,101 +181,9 @@ const PatientScheduledVisits: React.FC<PatientSearchProps> = ({ toggleSearchType
         </Button>
       </div>
 
-      <div className={styles.row}>
-        <p className={styles.heading}>{t('recentScheduledVisits', { count: recentVisits.length })} </p>
-        <TileGroup name="tile-group" defaultSelected="forever" className="trigger-tile">
-          {recentVisits.map((visit, index) => (
-            <RadioTile
-              id={visit.id}
-              value={visit.id}
-              key={visit.id}
-              className={styles.visitTile}
-              onClick={() => {
-                setShowRecentPriority(true);
-                setRecentVisitsIndex(index);
-              }}>
-              <div className={styles.helperText}>
-                <p className={styles.primaryText}>{visit.visit_type}</p>
-                <p className={styles.secondaryText}>
-                  {' '}
-                  {formatDatetime(parseDate(visit?.visit_date))} · {visit.clinic}{' '}
-                </p>
-                {showRecentPriority && index == recentVisitsIndex ? (
-                  <ContentSwitcher
-                    size="sm"
-                    className={styles.prioritySwitcher}
-                    onChange={({ index }) => {
-                      setRecentPrioritySwitcherValue(index);
-                    }}>
-                    <Switch
-                      name={priority.NOT_URGENT}
-                      text={t('notUrgent', 'Not Urgent')}
-                      value={recentPrioritySwitcherValue}
-                    />
-                    <Switch
-                      name={priority.PRIORITY}
-                      text={t('priority', 'Priority')}
-                      value={recentPrioritySwitcherValue}
-                    />
-                    <Switch
-                      name={priority.EMERGENCY}
-                      text={t('emergency', 'Emergency')}
-                      value={recentPrioritySwitcherValue}
-                    />
-                  </ContentSwitcher>
-                ) : null}
-              </div>
-            </RadioTile>
-          ))}
-        </TileGroup>
-      </div>
+      <PatientRecentScheduledVisits recentVisits={recentVisits} isLoading={isLoading} />
+      <PatientFutureScheduledVisits futureVisits={futureVisits} isLoading={loading} />
```

Comment:
Basically, these two components are duplicating code, achieving almost identical results, you can refactor it to something like 


```
<ScheduledVisits visitType='' visits={[]}/>
```

where you pass the visit type as the title for that visit either `recent` or `future` and visits array prop which can either be `recentVisits` or `futureVisits`